doc/source/dev/development_environment.rst

```diff
@@ -211,10 +211,20 @@ Debugging
 ---------
 
 Another frequently asked question is "How do I debug C code inside NumPy?".
-The easiest way to do this is to first write a Python script that invokes the C
-code whose execution you want to debug. For instance ``mytest.py``::
+First you must obtain the correct version of gdb (often the default on
+Linux), which requires the embedded Python version higher than 3.6.
+To see which version of Python is embedded inside your gdb, do the following::
```

Comment:
I am able to run gdb both with 3.5 and 3.8. I think we should avoid mentioning python version here.

Reply:
I tried python2.7 and failed to debug, should I just mention python3 without minor version?

Reply:
I agree with @anirudh2290  - the Python version embedded in gdb feels like a gdb-configuration-specific detail that we don't need to get entrenched on. IMO it's easiest to simply leave the version info out.

If users end up having trouble with a gdb-Python interaction, they'll be looking to docs other than NumPy's to solve it anyways.

Reply:
ok then, specific python version removed.